Formula/w/wasmedge.rb

```diff
@@ -17,16 +17,12 @@ class Wasmedge < Formula
   end
 
   depends_on "cmake" => :build
-  depends_on "ninja" => :build
```

Comment:
why is ninja removed?

Reply:
Our preference in Homebrew/core has been Unix Makefiles for CMake unless there is a strong reason for Ninja (e.g. Unix Makefile generation is broken for specific repo).

Reply:
aha, I was not aware that makefile over ninja preference, any docs links?

Reply:
I don't think it was explicitly documented but I recall some comments/discussions before on this.

Our default choice in `brew create` can be considered the guideline, i.e. `cmake` will use `make` (official default) and `meson` will use `ninja` (official default). And perhaps the majority of existing formulae.

A large portion of Ninja performance is from incremental builds which isn't useful in Homebrew builds and we avoid the extra dependency.

Reply:
yeah, that actually makes sense to me 👍